packages/schemas/tables/organization_jit_roles.sql

```diff
@@ -0,0 +1,14 @@
+/* init_order = 2 */
+
+/** The organization roles that will be automatically provisioned to users when they join an organization through JIT. */
+create table organization_jit_roles (
```

Comment:
Also need to add the new table to the DB seed cli command?

Reply:
in core we can just add the sql file